From 7f0a8e1e68cfd4a44f48ad32bc1e19a42a8c894b Mon Sep 17 00:00:00 2001
From: Julian Gercke <julian.gercke@gmx.de>
Date: Mon, 22 Feb 2021 14:05:59 +0100
Subject: [PATCH] namespace changes

---
 validation/sparql/SPARQLPrefixHandler.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/validation/sparql/SPARQLPrefixHandler.py b/validation/sparql/SPARQLPrefixHandler.py
index ab8c854..5e76426 100644
--- a/validation/sparql/SPARQLPrefixHandler.py
+++ b/validation/sparql/SPARQLPrefixHandler.py
@@ -6,6 +6,10 @@ prefixes = {
     "dbr": "<http://dbpedia.org/resource/>",
     "yago": "<http://dbpedia.org/class/yago/>",
     "foaf": "<http://xmlns.com/foaf/0.1/>",
+    "test1": "<http://example.org/testGraph1#>",
+    "test2": "<http://example.org/testGraph2#>",
+    "test3": "<http://example.org/testGraph3#>",
+    "test4": "<http://example.org/testGraph4#>",
     "": "<http://example.org/>"
 }
 
-- 
2.25.1

